src/rdkafka_broker.c

```diff
@@ -76,6 +76,7 @@
 
 
 static const int rd_kafka_max_block_ms = 1000;
+static const unsigned short HOLBSTR_MAX_SIZE = 256;
```

Comment:
Why is this change needed?
That buffer size is local to the timeout_scan() function and only used once, so no need for a predefined value.

Reply:
The important thing here is the change in the value to avoid compilation warning. Regarding the usage of a constant, I just prefer using constants that define what is a buffer for (to avoid magic numbers). However, if not necessary, I can return it to where it was.

Reply:
Ah, I see. 
Yes, please remove the constant and just the use the value directly.

Reply:
Sure. Done.